snapcraft/tests/test_commands.py

```diff
@@ -14,25 +14,15 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import fixtures
 
-from unittest import mock
-
-from snapcraft import help
+# Snapcraft modules
 from snapcraft import tests
+from snapcraft import commands
```

Comment:
This one doesn't appear in pep8, it's just how the majority of the canonical code I've seen does it. So it's like a soft style guide, we can override that with our own style on this project if you prefer.